Documentation/kubernetes/install.rst

```diff
@@ -286,44 +286,174 @@ Deploying the DaemonSet
     .. parsed-literal::
 
       $ wget \ |SCM_WEB|\/examples/kubernetes/1.7/cilium.yaml
-      $ vim cilium.yaml
-      [adjust the etcd address]
 
   .. group-tab:: K8s 1.8
 
     .. parsed-literal::
 
       $ wget \ |SCM_WEB|\/examples/kubernetes/1.8/cilium.yaml
-      $ vim cilium.yaml
-      [adjust the etcd address]
 
   .. group-tab:: K8s 1.9
 
     .. parsed-literal::
 
       $ wget \ |SCM_WEB|\/examples/kubernetes/1.9/cilium.yaml
-      $ vim cilium.yaml
-      [adjust the etcd address]
 
   .. group-tab:: K8s 1.10
 
     .. parsed-literal::
 
       $ wget \ |SCM_WEB|\/examples/kubernetes/1.10/cilium.yaml
-      $ vim cilium.yaml
-      [adjust the etcd address]
 
   .. group-tab:: K8s 1.11
 
     .. parsed-literal::
 
       $ wget \ |SCM_WEB|\/examples/kubernetes/1.11/cilium.yaml
-      $ vim cilium.yaml
-      [adjust the etcd address]
 
+Adjusting the ConfigMap
+-----------------------
 
-After configuring the ``cilium`` `ConfigMap` it is time to deploy it using
-``kubectl``:
+After downloading the ``cilium.yaml`` file, open it with your text editor and
+change the `ConfigMap` inside of it with the necessary changes based on the
```

Comment:
Remove " inside of it with the necessary changes"

Reply:
Done